express/scripts/scripts.js

```diff
@@ -654,6 +654,7 @@ export function getCurrency(locale) {
     bo: 'USD',
     do: 'USD',
     hr: 'EUR',
+    in: 'INR',
```

Comment:
Looks like the 'in' key is already in there.

Reply:
you are right @toka1300 , somehow it did not catch my eyes.. removed it, thanks